Fabric Ticketing Customer Facing Document.md

```diff
@@ -0,0 +1,141 @@
+# Introduction
```

Comment:
go ahead and remove this file now. You have a word document backup anyways right?
Make sure this is the one in the top folder that you are deleting. Don't delete the one in your content folder.

Reply:
Deleted the file.